CMakeLists.txt

```diff
@@ -119,9 +123,23 @@ add_library(freetype-gl SHARED
     ${FREETYPE_GL_SRC}
     ${FREETYPE_GL_HDR}
 )
-target_link_libraries( freetype-gl
-${FREETYPE_LIBRARIES}
-)
+if(NOT DEFINED ANDROID_ABI)
```

Comment:
Are you requesting no call to target_link or simply no branching between the two build variants.  I originally had fewer links listed in the linux branch.